curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6551f22653e6ce6e4f1d1370.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 6551f22653e6ce6e4f1d1370
+title: Step 10
+challengeType: 20
+dashedName: step-10
+---
+
+# --description--
+
+Use the `type()` function and print the data type of your `shift` variable.
+
+# --hints--
+
+You should pass `shift` to your `type()` function.
```

Comment:
```suggestion
You should pass `shift` to the `type()` function.
```

Reply:
I would suggest changing this to "the" to emphasize that the type() function is a built-in function instead of a user-defined function.